src/features/click-to-load/components/ctl-placeholder-blocked.js

```diff
@@ -0,0 +1,187 @@
+import { html } from '../../../dom-utils'
+import css from '../assets/ctl-placeholder-block.css'
+import { logoImg as daxImg } from '../ctl-assets'
+
+export class DDGCtlPlaceholderBlocked extends HTMLElement {
```

Comment:
Could you add a comment to explain what this class is for? I know it's kind of in the name, but I'm thinking if this is only for non-preview placeholders and only for mobile it would be nice to explain.

Reply:
Or maybe just add some text to the start of the comment for the constructor, to explain how it's used and what it's used for?

Reply:
done 👍